### ISSUE TITLE
Replacing token by suggestion is not a good idea

### ISSUE THREAD

**Issue by sammydeprez** [MicrosoftDocs/xamarin-docs#1394]
By using the replacement function you might replace tokens that are correct.
Let's say that you have in a sentence 2 times the token 'their'
And only one needs to be corrected by 'there' then both will be replaced by the 'Replace' function
What would be incorrect.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: e4a39e8c-61c2-834f-bdb1-88e74ada6fbb
* Version Independent ID: e6a5503b-e676-6d08-74de-3aa3ec8816ba
* Content: [Spell Checking Using the Bing Spell Check API - Xamarin](https://docs.microsoft.com/en-us/xamarin/xamarin-forms/data-cloud/cognitive-services/spell-check)
* Content Source: [docs/xamarin-forms/data-cloud/cognitive-services/spell-check.md](https://github.com/MicrosoftDocs/xamarin-docs/blob/live/docs/xamarin-forms/data-cloud/cognitive-services/spell-check.md)
* Product: **xamarin**
* GitHub Login: @davidbritch
* Microsoft Alias: **dabritch**

**Comment by conceptdev** [comment]
Good point - we just did that for simplicity. Adding a note to warn other implementors, thanks!
